File: php/difference_of_squares.php
<?php

function squareOfSums($n) {
   return pow(($n * ($n + 1) / 2),2);
}

function sumOfSquares($n) {
    return pow($n,3)/3 + pow($n,2)/2 + $n/6;
}

function difference($n) {
   return squareOfSums($n) - sumOfSquares($n);
}

File: python/difference_of_squares.py
from math import pow 

def square_of_sum(num):
    return pow(num*(num + 1)/2 ,2)

def sum_of_squares(num):
    return sum (x ** 2 for x in range(num + 1))

def difference(num):
    return square_of_sum(num) - sum_of_squares(num)

